browser-testing.md

```diff
@@ -1066,6 +1112,32 @@ The `waitForKey` method opens the current page URL in the default web browser an
 $page->waitForKey(); // Useful for debugging
 ```
 
+<a name="wait-for-url"></a>
+### waitForURL
```

Comment:
Can you update this PR in general? This method is no longer present on the release: https://github.com/pestphp/pest-plugin-browser/compare/v4.0.3...4.x